src/scripts/main.js

```diff
@@ -1,3 +1,90 @@
 'use strict';
 
-// write code here
+const MIN_COUNT = 2;
+const MAX_COUNT = 10;
+
+const table = document.querySelector('.container .field tBody');
```

Comment:
The selector '.container .field tBody' might not correctly select the intended table body element. Ensure that the HTML structure matches this selector, or adjust the selector to accurately target the desired element.